### ISSUE TITLE
Can't install node/6.1.0@silkedit/stable package in Windows

### ISSUE THREAD

**Issue by shinichy** [conan-io/conan#582]
After upgrading to 0.14, I can't install node/6.1.0@silkedit/stable package in Windows.
`conan install` command says 'Already installed' although it's not.
There's .conan_link in package directory after installation.

os: Windows 10 x64 (Japanese)

```
C:\Users\shinichi>conan install node/6.1.0@silkedit/stable -s arch=x86_64 -s compiler="Visual Studio" -s compiler.version=14
node/6.1.0@silkedit/stable: Not found, looking in remotes...
node/6.1.0@silkedit/stable: Trying with 'conan.io'...
Downloading conanmanifest.txt
[==================================================]
Downloading conanfile.py
[==================================================]
Downloading conan_export.tgz
[==================================================]
Requirements
    node/6.1.0@silkedit/stable from conan.io
Packages
    node/6.1.0@silkedit/stable:c85f9b402dd4d46acdf074e1c63b768a41181d7a

node/6.1.0@silkedit/stable: Already installed!

C:\Users\shinichi>dir .conan\data\node\\6.1.0\silkedit\stable\package\c85f9b402dd4d46acdf074e1c63b768a41181d7a
 ドライブ C のボリューム ラベルがありません。
 ボリューム シリアル番号は 0E81-321B です

 C:\Users\shinichi\.conan\data\node\6.1.0\silkedit\stable\package\c85f9b402dd4d46acdf074e1c63b768a41181d7a のディレクトリ

2016/10/20  22:05    <DIR>          .
2016/10/20  22:05    <DIR>          ..
2016/10/20  22:05                19 .conan_link
               1 個のファイル                  19 バイト
               2 個のディレクトリ  104,055,754,752 バイトの空き領域
```

**Comment by memsharded** [comment]
Hi!
Yes, we are aware of it, it is the same as: #581 

It is a short_paths issue on win, we are on it, thanks for reporting!

**Comment by memsharded** [comment]
Released 0.14.1, that should solve it, please try and give feedback. Thanks!

**Comment by shinichy** [comment]
It worked! Thanks for quick fix!
